src/runner/Resources.resx

```diff
@@ -116,8 +116,11 @@
     <value>Exit</value>
     <comment>Exit as a verb, as in Exit the application</comment>
   </data>
+  <data name="HIDE_TRAY_ICON_MENU_TEXT" xml:space="preserve">
+	<value>Hide Icon</value>
+  </data>
   <data name="SUBMIT_BUG_TEXT" xml:space="preserve">
-    <value>Report bug</value>
+    <value>Report Bug</value>
   </data>
   <data name="BUGREPORT_SUCCESS" xml:space="preserve">
     <value>Bug report .zip file has been created on your Desktop.</value>
```

Comment:
Not the most important, but is this the best grammar?

Reply:
This was from an existing commit, I just capitalized "bug". It should probably read "Report a bug" if we are sticking to sentences and not 'titles', yes.

Reply:
What I meant was, there are a few possibilities to indicate passed tense. Like "was created", "is created"... But if you label it off-topic I understand.